### ISSUE TITLE
Stop encoding keccak arguments with latin-1

### ISSUE THREAD

**Issue by carver** [ethereum/eth-utils#48]
### What was wrong?

`eth_utils.keccak()` is encoding text arguments with `latin-1`. This is different from behavior of other eth libraries.

This is a sub-task of #36 

See https://gist.github.com/holiman/6f57cb5b082b6712b9e8c19fe48ca960#gistcomment-2332077

### How can it be fixed?

One of:

- only accept bytes as argument
- convert text to utf-8
- **Edit:** use the `(primitive=None, texstr=None, hexstr=None)` argument signature approach -- *preferred*

Probably requires a major version bump

**Comment by pipermerriam** [comment]
How about changing the signature to match the convention we've been establishing elsewhere.

```
def keccak(primative=None, textstr=None, hexstr=None)
    ...
```

**Comment by carver** [comment]
Closed by #57
